http.go

```diff
@@ -56,6 +56,10 @@ func (s *Server) Handler() http.Handler {
 		w.Write([]byte("ok\n"))
 	})
 
+	for endpoint, customHandler := range s.HttpCustomHandlers {
+		mux.HandleFunc(pat.Get(endpoint), customHandler)
```

Comment:
Do we want to support HTTP methods besides `GET`?

Reply:
IMO - No. The POST/PUT/DELETE endpoints should be a more deliberate operation because it can affect state of the Veneur Server.